storage_backend_ftp/components/ftp_adapter.py

```diff
@@ -150,8 +151,8 @@ def move_files(self, files, destination_path):
                     _logger.debug("destination %s is free", dest_file_path)
                 if result:
                     client.delete(dest_file_path)
-                # Move the file
-                client.rename(ftp_file, dest_file_path)
+                # Move the file using absolute filepaths
+                client.rename(fp(ftp_file), fp(dest_file_path))
```

Comment:
watch out: existing services could rely on the fact that the fullpath is passed already.
I would improve `_fullpath` to check if the base path is already there or not.

Reply:
@simahawk right, I'll update `_fullpath()` too to check that out